xdn-next-example/xdn/cache.js

```diff
@@ -0,0 +1,43 @@
+export const SSR_CACHE_HANDLER = ({ removeUpstreamResponseHeader, cache }) => {
+  removeUpstreamResponseHeader('cache-control')
+  cache({
+    browser: {
+      maxAgeSeconds: 0,
+    },
+    edge: {
+      maxAgeSeconds: 60 * 60 * 24 * 365 * 10,
+      staleWhileRevalidateSeconds: 60 * 60 * 24,
+    },
+  })
+}
+
+export const API_CACHE_HANDLER = ({ setResponseHeader, removeUpstreamResponseHeader, cache, proxy }) => {
+  removeUpstreamResponseHeader('cache-control')
+  cache({
+    browser: {
+      maxAgeSeconds: 0,
+      serviceWorkerSeconds: 60 * 60 * 24,
+    },
+    edge: {
+      maxAgeSeconds: 60 * 60 * 24 * 365 * 10,
+      staleWhileRevalidateSeconds: 60 * 60 * 24,
+    },
+  })
+  proxy('api')
+  setResponseHeader('content-type', 'application/json')
```

Comment:
You should set this content-type header on the API app. That way you don't need to set it in each example app.